### PR TITLE
Extract common dashboard functionality

### PR DESCRIPTION
**Pull request by rawagner**
web-ui-components PR https://github.com/kubevirt/web-ui-components/pull/439

@gnehapk @AyushAmbastha @cloudbehl Can you please try this with the web-ui-components PR on environment with Cepth to make sure i didnt break anything ? Thank you

### CODE REVIEW

**Comment by cloudbehl** [comment]
@rawagner Ack! will test it and let you know.

**Comment by cloudbehl** [comment]
@rawagner Tested the PR. Looks good to me!

Screenshot from 2019-05-07 17-25-25

**Comment by rawagner** [comment]
@cloudbehl Thank you!

**Comment by cloudbehl** [comment]
@rawagner https://github.com/kubevirt/web-ui/pull/361/files#diff-f5fb6fd19d06f558ee1919e253b61608R154 was breaking earlier when I pulled it but later looks like you fixed it.

File: frontend/public/kubevirt/components/dashboards/dashboards.jsx [hunk 1]
```diff
@@ -2,6 +2,49 @@ import React from 'react';
 import { ClusterOverview } from '../cluster/cluster-overview';
```
Comment (mareklibra) [inline frontend/public/kubevirt/components/dashboards/dashboards.jsx]: When refactoring, I would split content of this dashboard.jsx to multiple files:
- index.jsx for re-export
- utils.jsx (or similar) for functions
- dashboards.jsx for the component

Imports in other files should be updated accordingly.

Comment (rawagner) [reply frontend/public/kubevirt/components/dashboards/dashboards.jsx]: done

**Pull request merged**